taniumrest_connector.py

```diff
@@ -940,94 +988,119 @@ def _handle_get_question_results(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _sanitize_sensors(self, action_result, obj, param_list):
+        if isinstance(obj, list):
```

Comment:
It looks like we're loading the full sensor version to the obj, right?
In that case, would "load_full_sensors_to_obj" be better?

Reply:
Refactored the method name to _ load_full_sensors_to_obj.